src/vs/platform/native/electron-main/nativeHostMainService.ts

```diff
@@ -748,6 +750,12 @@ export class NativeHostMainService extends Disposable implements INativeHostMain
 		return app.runningUnderARM64Translation;
 	}
 
+	@memoize
```

Comment:
Not sure about this property in this service. The idea in this service is to provide methods to the renderer. And if its a static property it would better be suited in environment service.